src/components/EpisodeTableCell.tsx

```diff
@@ -18,67 +19,85 @@ type Props = {
 export const EpisodeTableCell = (props: Props) => {
   const { pubDate, description, title = 'untitled episode', handleMorePress,
   handleNavigationPress, moreButtonAlignToTop, podcastImageUrl, podcastTitle } = props
+  const [globalTheme] = useGlobal('globalTheme')
 
   const moreButtonStyles = [styles.moreButton]
   if (moreButtonAlignToTop) moreButtonStyles.push(styles.moreButtonAlignToTop)
 
+  const innerTopView = (
+    <View style={styles.innerTopView}>
+      {
+        podcastImageUrl &&
```

Comment:
I will leave the note here but its for everywhere. Wherever you do `potentialString && <component>` it will crash on Android. Android cannot handle strings inside views. you'll need to coerce it to a bool `!!potentialString`

Reply:
Ok. So just strings? I won't have to coerce functions? Or should I just use !! whenever I do a truthy test in a view?